Sparkle/SUBinaryDeltaCreate.m

```diff
@@ -574,13 +586,28 @@ BOOL createBinaryDelta(NSString *source, NSString *destination, NSString *patchF
 
     xar_close(x);
 
-    unlink([patchFile fileSystemRepresentation]);
-    link([temporaryFile fileSystemRepresentation], [patchFile fileSystemRepresentation]);
-    unlink([temporaryFile fileSystemRepresentation]);
-
+    NSFileManager *filemgr;
+    filemgr = [NSFileManager defaultManager];
+    
+    [filemgr removeItemAtPath: patchFile error: NULL];
```

Comment:
This could be simplified to use `moveItemAtPath`. Also some stray NSLogs are left

Reply:
Will do.